packages/backend/src/services/NotificationsService/NotificationsService.ts

```diff
@@ -0,0 +1,38 @@
+import { Notification } from '@lightdash/common';
+import { NotificationsModel } from '../../models/NotificationsModel/NotificationsModel';
+
+type Dependencies = {
+    notificationsModel: NotificationsModel;
+};
+
+export class NotificationsService {
+    notificationsModel: NotificationsModel;
+
+    constructor({ notificationsModel }: Dependencies) {
+        this.notificationsModel = notificationsModel;
+    }
+
+    async getDashboardCommentNotifications(
+        userUuid: string,
+    ): Promise<Notification[]> {
+        return this.notificationsModel.getDashboardCommentNotifications(
+            userUuid,
+        );
+    }
+
+    async markNotificationAsRead(notificationUuid: string) {
+        return this.notificationsModel.markNotificationAsRead(notificationUuid);
+    }
+
+    async createDashboardCommentNotification(
```

Comment:
instead of this service, I'd call directly `this.notificationsModel.createDashboardCommentNotification` after create comment on the comment service.